mne/tests/common.py

```diff
@@ -0,0 +1,48 @@
+# Authors: Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from numpy.testing import assert_allclose
+
+from .. import pick_types, Evoked
+from ..io import _BaseRaw
```

Comment:
Hmm... this isn't really a *test*, though, it's a set of functions used *by tests* (but in theory could be used other places).